mlflow_elasticsearchstore/elasticsearch_store.py

```diff
@@ -315,6 +334,30 @@ def _get_orderby_clauses(self, order_by_list: List[str], s: Search) -> Search:
         s = s.sort(*sort_clauses)
         return s
 
+    def _columns_to_whitelist(self, columns_to_whitelist: List[str]) -> List[Q]:
```

Comment:
I would rename it something like `_get_column_to_whitelist_query` or `_get_columns_to_whitelist_contraints`